rules/architecture-diagram/rule.md

```diff
@@ -162,21 +160,31 @@ Maintain standards to keep your diagrams consistent:
 ![Figure: Good example - SSW People (a Static Site - Gatsby and React with Dynamics 365 and SharePoint Online) - you can just as easily create colorful, engaging diagrams suitable for all of your project stakeholders](SSW.People-Architecture-Diagram.png)
 :::
 
-### Tip #9: Where to store Diagrams?
+### Tip #10: Exporting diagrams
+
+To export your diagram, go to *File* | *Export as* | *PNG*
+
+Make sure to tick *Include a copy of my diagram* is selected. This will let you open your diagram and edit it again when you import the PNG file into draw.io.
+
+To make it easy to tell that you can open the file in draw.io, ensure that the file extension is `.drawio.png`. 
+
+Read more in the docs - https://www.drawio.com/doc/faq/export-to-png
+
+### Tip #11: Where to store Diagrams?
 
 Standardizing where your organisation stores architecture diagrams ensures a consistent experience among developers. Therefore store your architecture diagrams in the repo **docs**\ folder. Additionally, the \README.md (in the root) should have a link and an embedded image of the high-level architecture diagram (from the **docs**\\* folder).
 
 **Note:** If you have a Wiki, for visibility add an architecture diagram page and embed the images from the **docs**\\* folder.
 
-### Tip #10: Use Azure Architecture Center
+### Tip #12: Use Azure Architecture Center
 
 [Azure Architecture Center](https://docs.microsoft.com/en-us/azure/architecture/) is the best tool to help you figure out the pieces you need for an architecure diagram - see [SSW.Rules | Do you use Azure Architecture Center](/azure-architecture-center)
 
-### Alternatives to Diagrams.net
+### Alternatives to draw.io
 
 **Miro (by Adobe)**
 
-[Miro](https://miro.com/) is an online tool designed primarily for whiteboard-style collaboration. It is very easy to use and optimised for this purpose. As a diagramming tool, it is lacking in features compared to Diagrams.net, but it can be used to create simple diagrams.
+[Miro](https://miro.com/) is an online tool designed primarily for whiteboard-style collaboration. It is very easy to use and optimised for this purpose. As a diagramming tool, it is lacking in features compared to draw.io, but it can be used to create simple diagrams.
```

Comment:
```suggestion
[Miro](https://miro.com/) is an online tool designed primarily for whiteboard-style collaboration. It is very easy to use and optimised for this purpose. As a diagramming tool, it is lacking in features compared to Draw.io, but it can be used to create simple diagrams.
```

Reply:
lowercasing all references for consistency